main.scss

```diff
@@ -1,4 +1,12 @@
-@import "node_modules/govuk-frontend/all";
+// @import "node_modules/govuk-frontend/objects/all";
+// @import "node_modules/govuk-frontend/core/all";
+
+@import "node_modules/govuk-frontend/settings/all";
+@import "node_modules/govuk-frontend/tools/all";
+@import "node_modules/govuk-frontend/helpers/all";
+
+// @import "node_modules/govuk-frontend/components/checkboxes/checkboxes";
+// @import "node_modules/govuk-frontend/components/breadcrumbs/breadcrumbs";
```

Comment:
Is there a reason we've commented these out? could they just not be there?

Reply:
They're removed in 0cb8a2e6e10efffb0b2d4de4a05a9cb334c4cdce. I intent to merge them so they'll fade away.